src/styles/blocks/snake.scss

```
@@ -0,0 +1,28 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($box-height, 1fr));
```

It's not only heigth, it's also width.
```suggestion
  grid-template-columns: repeat(auto-fill, minmax($box-side, 1fr));
```